src/App.js

```diff
@@ -9,6 +9,7 @@ class App extends React.Component {
   state = {
     hello: 1,
```

Comment:
If I recall `hello` and `bye` were just dummy variables for explanation. They can be removed.